openedx/core/djangoapps/course_groups/management/commands/remove_users_from_multiple_cohorts.py

```diff
@@ -0,0 +1,56 @@
+"""
+Script for removing users with multiple cohorts of a course from cohorts
+to ensure user's uniqueness for a course cohorts
+"""
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from openedx.core.djangoapps.course_groups.models import CourseUserGroup
+
+
+class Command(BaseCommand):
+    """
+    Remove users with multiple cohorts of a course from all cohorts
+    """
+    help = 'Remove all users from multiple cohorts (except one) of each course'
+
+    def handle(self, *args, **options):
+        """
+        Execute the command
+        """
+        # Get entries of cohorts which have same user added multiple times for a single course
+        multiple_objects_cohorts = CourseUserGroup.objects.filter(group_type=CourseUserGroup.COHORT).\
+            values_list('users', 'course_id').annotate(user_count=Count('users')).filter(user_count__gt=1).\
+            order_by('users')
```

Comment:
Okay, I may not be reading this right. But can you add the following to the test:

1. Create user2.
2. Force add to one of the cohorts.
3. After running the command, check that user2 is still in the same cohort. You will need to pass `assign=False` to `get_cohorts` to ensure that they do not get assigned a cohort.